### ISSUE TITLE
RC Test: Resistance in a Wire 1.5.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#108]
@ariel-phet, @emily-phet, @terracoda, @phet-steele, Resistance in a Wire 1.5.0-rc.1 is ready for RC testing.

**[Link to sim](http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.5.0-rc.1/phet/resistance-in-a-wire_all_phet.html)**

**[Link to iFrame](http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.5.0-rc.1/phet/resistance-in-a-wire_all_iframe_phet.html)**

**[Test Matrix](https://docs.google.com/spreadsheets/d/1qNs8hbWvo19YKgn1ds_Y1AuV8N8k9tneGWaRB9N4plE/edit#gid=783558704)**

**Before beginning**, please familiarize yourself with how a screen reader works. Here is a page with information about PhET's supported screen readers and documentation about how to use them:
[Screen reader intro](https://www.colorado.edu/physics/phet/dev/html/jg-tests/reader-intro.html)

The simulation to test has labels and/or descriptions. They can be "seen" here:
**[Link to a11y view](http://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.5.0-rc.1/phet/resistance-in-a-wire_a11y_view.html)**

This view shows what is available to a screen reader user. The sim is on the left side in an iframe, while the right side has all of the accessible descriptions. Beneath the sim are real-time alerts that will be announced by the screen reader. Please make sure that the descriptions and alerts accurately describe the simulation at all times.

PhET supports the following platforms for accessibility so please test these, verifying that all content in the a11y view is readable with the virtual cursor and navigation strategies listed in the [Screen reader intro](https://www.colorado.edu/physics/phet/dev/html/jg-tests/reader-intro.html):
*(These should also be in the test matrix)*
- [x] JAWS with latest Windows, latest Firefox
- [x] NVDA with latest Windows, latest Firefox
- [x] VoiceOver with latest macOS, latest Safari

**Please also verify**
- [x] stringTest=double (all strings doubled)
- [x] stringTest=long (exceptionally long strings)
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (test passes if sim does not redirect, OK if sim crashes or fails to fully start)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Screenshot test
- [x] Does AR need to be notified for any LOL updates?

Accessibility strings that are not visible should not be translatable yet, do not worry if these do not change with string tests.

If any new issues are found, please note them in https://github.com/phetsims/resistance-in-a-wire/issues and reference this issue. 

**Critical screen reader information**
- JAWS might complain about Firefox Quantum. If that is the case, please test with https://www.mozilla.org/en-US/firefox/organizations/
- The screen reader **must** be turned on and reading before the browser is open for it to work. See https://github.com/phetsims/a11y-research/issues/90

@ariel-phet please assign and prioritize.

@phet-steele, this issue was made from a template for a11y RC tests. If you have any thoughts on this template, or on a11y testing in general, it would  be great to hear them.

**Comment by lmulhall-phet** [comment]
@jessegreenberg QA is done!

**Comment by terracoda** [comment]
Way to go team!

**Comment by jessegreenberg** [comment]
This RC branch will need to get the updates for  https://github.com/phetsims/scenery/issues/770
